lib/DxilContainer/DxilContainerAssembler.cpp

```diff
@@ -1829,50 +1768,26 @@ class DxilRDATWriter : public DxilPartWriter {
                                              TGSMInFunc[&function]);
             } else if (DXIL::CompareVersions(m_ValMajor, m_ValMinor, 1, 8) >
                        0) {
```

Comment:
Is this right or should it be `>=`? Maybe it's part of an experimental feature? 
```suggestion
            } else if (DXIL::CompareVersions(m_ValMajor, m_ValMinor, 1, 8) >=
                       0) {
```

Reply:
It's not included for 1.8, it's part of the experimental/potential next version.